### PR TITLE
Updated Hebrew translation for v2.0 & v2.2

### PR DESCRIPTION
**Pull request by orisraeli**
Translation credit: Or Israeli.
@david-levi care to review these changes?

### CODE REVIEW

File: he.lproj/Localizable.strings [hunk 1]
```diff
@@ -161,73 +161,73 @@
...
-"WELCOME_NEWFEATURES_2_AIRPLAY" = "AirPlay toolbar controls";
-"WELCOME_NEWFEATURES_2_ALERTS" = "Less-intrusive alerts";
-"WELCOME_NEWFEATURES_2_NATIVESCALING" = "Native pixel scaling";
-"WELCOME_NEWFEATURES_2_APPLESILICON" = "Supports Apple Silicon";
+"WELCOME_NEWFEATURES_2_DESIGN" = "מראה ותחושה מודרניים";
+"WELCOME_NEWFEATURES_2_SLEEPTIMER" = "שעון עצר לשינה";
+"WELCOME_NEWFEATURES_2_AIRPLAY" = "שליטה על AirPlay";
+"WELCOME_NEWFEATURES_2_ALERTS" = "התראות פחות מפריעות";
+"WELCOME_NEWFEATURES_2_NATIVESCALING" = "תמיכה ברזולוציות מסך גבוהות";
+"WELCOME_NEWFEATURES_2_APPLESILICON" = "תמיכה ב- Apple Silicon";
```
Comment (david-levi) [inline he.lproj/Localizable.strings:186]: Remove the space after the hyphen.

File: he.lproj/Localizable.strings [hunk 2]
```diff
@@ -161,73 +161,73 @@
...
-"WELCOME_NEWFEATURES_2_DESIGN" = "Modern look & feel";
-"WELCOME_NEWFEATURES_2_SLEEPTIMER" = "Sleep Timer";
-"WELCOME_NEWFEATURES_2_AIRPLAY" = "AirPlay toolbar controls";
-"WELCOME_NEWFEATURES_2_ALERTS" = "Less-intrusive alerts";
-"WELCOME_NEWFEATURES_2_NATIVESCALING" = "Native pixel scaling";
-"WELCOME_NEWFEATURES_2_APPLESILICON" = "Supports Apple Silicon";
+"WELCOME_NEWFEATURES_2_DESIGN" = "מראה ותחושה מודרניים";
+"WELCOME_NEWFEATURES_2_SLEEPTIMER" = "שעון עצר לשינה";
+"WELCOME_NEWFEATURES_2_AIRPLAY" = "שליטה על AirPlay";
+"WELCOME_NEWFEATURES_2_ALERTS" = "התראות פחות מפריעות";
```
Comment (david-levi) [inline he.lproj/Localizable.strings:184]: I am not too satisfied about this string... That said, I do find it challenging to come up with something better.
How about: "התראות יותר מעודנות"? Just throwing it out there, your version works as well.

File: he.lproj/Localizable.strings [hunk 3]
```diff
@@ -161,73 +161,73 @@
...
 
-"WELCOME_NEWFEATURES_2_DESIGN" = "Modern look & feel";
-"WELCOME_NEWFEATURES_2_SLEEPTIMER" = "Sleep Timer";
-"WELCOME_NEWFEATURES_2_AIRPLAY" = "AirPlay toolbar controls";
-"WELCOME_NEWFEATURES_2_ALERTS" = "Less-intrusive alerts";
-"WELCOME_NEWFEATURES_2_NATIVESCALING" = "Native pixel scaling";
-"WELCOME_NEWFEATURES_2_APPLESILICON" = "Supports Apple Silicon";
+"WELCOME_NEWFEATURES_2_DESIGN" = "מראה ותחושה מודרניים";
+"WELCOME_NEWFEATURES_2_SLEEPTIMER" = "שעון עצר לשינה";
+"WELCOME_NEWFEATURES_2_AIRPLAY" = "שליטה על AirPlay";
```
Comment (david-levi) [inline he.lproj/Localizable.strings:183]: Why lose the "toolbar controls" portion? You can go with something like "סרגל כלים לשליטה ב-AirPlay".

File: he.lproj/Localizable.strings [hunk 4]
```diff
@@ -161,73 +161,73 @@
...
-"TOOLBAR_PREVIOUSBUTTON_TOOLTIP" = "Previous";
-"TOOLBAR_NEXTBUTTON_TOOLTIP" = "Next";
+"TOOLBAR_PREVIOUSBUTTON_TOOLTIP" = "הקודם";
+"TOOLBAR_NEXTBUTTON_TOOLTIP" = "הבא";
 
-"ALERT_UPGRADE" = "Upgrade";
+"ALERT_UPGRADE" = "שדרג";
 
-"MENU_APPLICATION_UPGRADE" = "Upgrade to Full Version…";
+"MENU_APPLICATION_UPGRADE" = "שדרוג לגירסא המלאה...";
```
Comment (david-levi) [inline he.lproj/Localizable.strings:214]: What do we use with all the other verbs? It should all either be "עיון", "שדרוג", "ניגון", or "עיין", "שדרג", "נגן", but not both.

File: he.lproj/Localizable.strings [hunk 5]
```diff
@@ -161,73 +161,73 @@
...
+"ERROR_ALERT_TITLE" = "שגיאה: %@";
 
-"LIBRARY_STATE_UNAUTHENTICATED" = "Not Logged In";
-"LIBRARY_STATE_QUOTA_EXCEEDED" = "Out of Space";
+"LIBRARY_STATE_UNAUTHENTICATED" = "לא מחובר";
+"LIBRARY_STATE_QUOTA_EXCEEDED" = "שטח האחסון מלא";
 
-"SIDEBAR_CATEGORY_DIRECTORY" = "Browse";
-"SIDEBAR_TITLE_DIRECTORY" = "Directory";
+"SIDEBAR_CATEGORY_DIRECTORY" = "עיון";
```
Comment (david-levi) [inline he.lproj/Localizable.strings:169]: What do we use with all the other verbs? It should all either be "עיון", "שדרוג", "ניגון", or "עיין", "שדרג", "נגן", but not both.

**Review by david-levi — COMMENTED** [review]
Overall looks great, most of my comments are merely suggestions.
Thanks for requesting my review 😄

**Comment by orisraeli** [comment]
Thanks @david-levi, I created a new commit with changes regarding your feedback.

**Comment by steventroughtonsmith** [comment]
Thanks for these, and for the edits! Give me a shout when I should integrate these changes, and I'll schedule them for a future update to Broadcasts. 😄

**Review by david-levi — APPROVED** [review]

**Comment by steventroughtonsmith** [comment]
Thanks! Will include these updates in Broadcasts v2.2.7 👍

**Pull request merged**